lib/Workflow/Condition/LazyOR.pm

```diff
@@ -37,10 +37,10 @@ sub evaluate {
     foreach my $cond ( @{$conditions} ) {
         my $result = $self->evaluate_condition( $wf, $cond );
         if ($result) {
-            return $result;
+            return Workflow::Condition::IsTrue->new("Got match in " . $cond );
         }
     }
-    return ''; # false
+    return Workflow::Condition::IsFalse->new();
```

Comment:
We seem to have a number of these "generic" return values -- where no message is defined. How about defining `$Workflow::Condition::TRUE` and `$Workflow::Condition::FALSE`.

Reply:
Can be done later.